core/src/main/scala/org/apache/spark/status/LiveEntity.scala

```diff
@@ -625,10 +626,19 @@ private class SchedulerPool(name: String) extends LiveEntity {
 
 }
 
-private object LiveEntityHelpers {
+private[spark] object LiveEntityHelpers {
 
   private val stringInterner = Interners.newWeakInterner[String]()
 
+  private def accuValuetoString(value: Any): String = value match {
+    case list: java.util.List[_] =>
+      if (list.size() > 5) {
```

Comment:
Would it be better if we add a config for this?

Reply:
I'm not sure if a config here is useful. Will we want to set more or less elements of collection accumulators in api or ui? cc @cloud-fan

Reply:
Why don't we just use existing configuration `spark.sql.debug.maxToStringFields`?

Reply:
Oops, it's not only SQL.

Reply:
yea. :)